etna/series/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+
+from wagtail.admin.panels import FieldPanel
+from wagtail.search import index
+
+from wagtailmetadata.utils import get_image_model_string
+
+
+class SeriesCategory(models.TextChoices):
+    RECORD_COLLECTION = "Record collection", "Record collection"
+
+
+class Series(index.Indexed, models.Model):
+    admin_name = models.CharField(
+        max_length=128,
+        help_text="The name of the series as it should be displayed in the CMS site.",
+    )
+    category = models.CharField(
+        choices=SeriesCategory.choices,
+        blank=True,
+    )
```

Comment:
Category will always be "Record Collection", and I think it'll be hard-coded. I might be wrong with this though, if there are other versions with other text here. Might be easier to remove it 👍

Reply:
Let's hard-code for now and iterate later if needed.